sdk/nodejs/eventgrid/zMixins.ts

```diff
@@ -132,9 +133,10 @@ export class EventGridCallbackSubscription<T> extends appservice.EventSubscripti
         const keys = pulumi.output(this.functionApp.getHostKeys());
         const key = keys.systemKeys["eventgrid_extension"];
         const url = pulumi.interpolate`https://${this.functionApp.defaultHostname}/runtime/webhooks/eventgrid?functionName=${name}&code=${key}`;
+        const liveUrl = url.apply(u => waitUntilEndpointIsUp(this, u));

```

Comment:
Is it guaranteed that this url endpoint is internet addressable?

Reply:
As of today, yes, Event Grid requires publicly accessible endpoints. Private endpoints require tricks like putting a proxy or a hybrid connection in between.